staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/strategy.go

```diff
@@ -149,7 +162,16 @@ func (customResourceStrategy) AllowUnconditionalUpdate() bool {
 
 // ValidateUpdate is the default update validation for an end user updating status.
 func (a customResourceStrategy) ValidateUpdate(ctx context.Context, obj, old runtime.Object) field.ErrorList {
-	return a.validator.ValidateUpdate(ctx, obj, old, a.scale)
+	var errs field.ErrorList
+	errs = append(errs, a.validator.ValidateUpdate(ctx, obj, old, a.scale)...)
+
+	// Checks the embedded objects. We don't make a difference between update and create for those.
+	if u, ok := obj.(*unstructured.Unstructured); ok {
+		gv := obj.GetObjectKind().GroupVersionKind().GroupVersion()
```

Comment:
why do we use GV here? the `structuralSchemas` map seems to be indexed by version only

Reply:
very good finding. Had fixes the create case, but not the udpate.